client/components/dashboard/dashboard-directive.html

```diff
@@ -43,9 +44,17 @@
               markdown-to-html="(widget.model.header_text || '') | stripHtml">
```

Comment:
The || '' is on the widgets, not the containers, to deal with a backwards compatibility issue for missing properties (it would show "undefined".  This should really be encapsulated in the version update script to add empty strings, but as this change wasn't introduced in this PR, I'll fix it in a separate one.